### ISSUE TITLE
Bluetooth active even if wifi is connected while in offline mode

### ISSUE THREAD

**Issue by andreialecu** [guh/nymea-networkmanager#8]
Cross posting from:
https://github.com/guh/berrylan/issues/9

Not sure where it is best to report this... Feel free to close the other one.

**Comment by andreialecu** [comment]
Thanks so much for fixing this @t-mon ! 

One followup question, I see that a 5 second delay was added.

Does it account for situations such as the wifi connection taking more than 5 seconds to connect? As you probably experienced, often there are cases when wifi takes several seconds to connect (for whatever unknown reason). Would this fix work in all cases?

**Comment by t-mon** [comment]
Hey andrei! So from what I saw, the issue was caused by a race condition. If network-manager connects while the bluetooth server starts, we ended up in the situation you described. Since this was a special case (during the startup procedure), this should not happen again, since the network-manager state will be evaluated once changed after the start procedure.
The build is almost trough, so it would be great if you could test it in order to verify.
The new version is 0.3.2 and should be available from the `main` section and `raspbian` section of the nymea repository. During the night the release will be published by the CI.

**Comment by andreialecu** [comment]
Confirmed that bluetooth is (properly) *not* broadcasting now!

1) I built the debian packages from source using the instructions in the README and installed them both.
2) Rebooted and confirmed that bluetooth wasn't broadcasting
3) Deleted saved connections from `/etc/NetworkManager/system-connections`
4) Rebooted again and now it was broadcasting and I was able to set it up properly using the Berrylan app. Also it properly stopped broadcasting after the setup.
5) Rebooted again, wifi connected automatically, bluetooth not broadcasting. As expected.

Thank you!
